include/seqan3/search/fm_index/bi_fm_index.hpp

```diff
@@ -93,27 +79,22 @@ class bi_fm_index
     //!\brief The type of the alphabet size of the underlying SDSL index.
     using sdsl_sigma_type = typename sdsl_index_type::alphabet_type::sigma_type;
 
-    //!\brief The type of the underlying FM index for the original text.
-    using fm_index_type = fm_index<text_layout{is_collection_}, sdsl_index_type>;
-
-    //!\brief The type of the underlying FM index for the reversed text.\if DEV \todo Change sampling behaviour. \endif
-    using rev_fm_index_type = fm_index<text_layout{is_collection_}, sdsl_index_type>;
+    //!\brief The type of the underlying FM index.
+    using fm_index_type = fm_index<alphabet_t, text_layout_mode, sdsl_index_type>;
```

Comment:
I think @cpockrandt had in mind that the reverse fm_index does something different.

> \if DEV \todo Change sampling behaviour. \endif

and thus the two different types.

Reply:
Good point. The `sdsl_index_type` might/should be different. I'll change it back accordingly.